hack/validate-migration.sh

```diff
@@ -111,12 +152,15 @@ check_apply_on_pipelines() {
     local updated=
     while read -r file_path; do
         cp "$file_path" "${file_path}.copy"
+        info "apply migration $migration_file to pipeline ${file_path}.copy"
```

Comment:
Does it make sense to have a separate variable for `${file_path}.copy"`?

Reply:
Updated. PTAL.